### ISSUE TITLE
在ubuntu22.04上构建cuda——libtorch镜像后运行car_detect样例工程时崩溃

### ISSUE THREAD

**Issue by JiangXiaoDong123** [modelbox-ai/modelbox#388]
在ubuntu22.04上构建cuda——libtorch镜像后运行car_detect样例工程时，使用webui的可视化界面启动后可以正常生成识别后的视频，看起来都正常，但是在删除任务时会导致程序崩溃，可以从何处调查呢，谢谢。错误日志如下：
[2026-03-14 15:56:28,537][ INFO][ffmpeg_video_demuxer.cc:154 ] Stream /root/ttttttt/src/graph/car_test_video.mp4 is end
[2026-03-14 15:56:28,538][ INFO][          node.cc:754 ] node: videodemuxer, sess id: f38198e8-5678-4e32-bfb0-b0c6d5f71c21, data ctx finished
[2026-03-14 15:56:28,794][ INFO][video_decoder_flowunit.cc:107 ] Video decoder finish
[2026-03-14 15:56:29,024][ INFO][          node.cc:754 ] node: videodecoder, sess id: f38198e8-5678-4e32-bfb0-b0c6d5f71c21, data ctx finished
[2026-03-14 15:56:29,112][ INFO][ editor_plugin.cc:661 ] template_dir:/usr/local/share/modelbox/project-template/project
[2026-03-14 15:56:29,385][ INFO][          node.cc:754 ] node: yolox_post, sess id: f38198e8-5678-4e32-bfb0-b0c6d5f71c21, data ctx finished
[2026-03-14 15:56:29,585][ INFO][          node.cc:754 ] node: videoencoder, sess id: f38198e8-5678-4e32-bfb0-b0c6d5f71c21, data ctx finished
[2026-03-14 15:56:29,585][ INFO][       session.cc:121 ] session f38198e8-5678-4e32-bfb0-b0c6d5f71c21 is over, running session count 0
[2026-03-14 15:56:29,585][ INFO][session_context.cc:44  ] session context finish se id:f38198e8-5678-4e32-bfb0-b0c6d5f71c21
[2026-03-14 15:57:00,411][ INFO][flow_scheduler.cc:459 ] shutdown flow scheduler.
[2026-03-14 15:57:00,412][ERROR][flow_scheduler.cc:438 ] the scheduler caught an error : Stop operation
[2026-03-14 15:57:00,418][ INFO][   job_manager.cc:82  ] delete job : ttttttt
[2026-03-14 15:57:00,518][ INFO][   cuda_memory.cc:91  ] Join release stream thread start
[2026-03-14 15:57:00,518][ INFO][   cuda_memory.cc:93  ] Release stream thread stop
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f6754003cd0         [R9]=0x0000000017c51abf         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f676effdb58        [R15]=0x00007f6851b97e00
[RDI]=0x00007f67540009b0        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f676effd9d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x0000000000000004        [RCX]=0x00007f676effcd98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f68ac14b190         [R9]=0x0000000071f76375         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f67ad7acb58        [R15]=0x00007f6851bbfe00
[RDI]=0x00007f68ac0008e0        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f67ad7ac9d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x0000000000000004        [RCX]=0x00007f67ad7abd98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f6890007770         [R9]=0x000000006ec84d0a         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f68ce017b58        [R15]=0x00007f6851bcfe00
[RDI]=0x00000007f6890007        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f68ce0179d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x00007f6890000060        [RCX]=0x00007f68ce016d98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f678c0036f0         [R9]=0x0000000014960454         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f6793fffb58        [R15]=0x00007f6851bc7e00
[RDI]=0x00007f678c0009b0        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f6793fff9d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x0000000000000004        [RCX]=0x00007f6793ffed98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f6774006100         [R9]=0x00000000b734a533         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f6790ff9b58        [R15]=0x00007f6851bb7e00
[RDI]=0x00007f67740009b0        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f6790ff99d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x0000000000000004        [RCX]=0x00007f6790ff8d98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f68c000d2b0         [R9]=0x000000009f1087d0         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f68dd3fcb58        [R15]=0x00007f6851b9fe00
[RDI]=0x00007f68c00008e0        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f68dd3fc9d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x0000000000000004        [RCX]=0x00007f68dd3fbd98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f674c001700         [R9]=0x000000008a1b80d8         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f68c4ff9b58        [R15]=0x00007f6851b87e00
[RDI]=0x00007f674c0009b0        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f68c4ff99d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x0000000000000004        [RCX]=0x00007f68c4ff8d98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: 11, Addr: 0x7f69dc068010, Code: 1, Caused by: 
[2026-03-14 15:57:00,543][ERROR][          main.cc:98  ] CPU Register Info:
[R8]=0x00007f67a4005680         [R9]=0x00000000e44dc98e         [R10]=0x0000000000000000        [R11]=0x0000000000000000
[R12]=0x00007f69f3a46ae8        [R13]=0x0000000000000004        [R14]=0x00007f67ab7a8b58        [R15]=0x00007f6851bdfe00
[RDI]=0x00000007f67a4005        [RSI]=0x00007f69f3a46b68        [RBP]=0x00007f67ab7a89d8        [RBX]=0x00007f69dc068010
[RDX]=0x0000000000000003        [RAX]=0x00007f67a4000060        [RCX]=0x00007f67ab7a7d98        [RSP]=0x00007f69dc068010
[RIP]=0x0000000000010202        [RFLAGS]=0x00007f69f38cf453
[2026-03-14 15:57:00,543][ERROR][          main.cc:93  ] Segment fault, Signal: Segmentation fault (core dumped)

**Comment by JiangXiaoDong123** [comment]
@pymumu 求助，现在使用的 是4060ti显卡，cuda版本12.2，libtorch2.10

[root@72e1620e4aa3 modelbox]$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2023 NVIDIA Corporation
Built on Tue_Aug_15_22:02:13_PDT_2023
Cuda compilation tools, release 12.2, V12.2.140
Build cuda_12.2.r12.2/compiler.33191640_0

[root@72e1620e4aa3 modelbox]$ python3 -c "import torch; print(torch.__version__)"
2.10.0+cu128

**Comment by JiangXiaoDong123** [comment]
释放的时候有顺序bug，通过ai修复了
